gammapy/irf/energy_dispersion.py

```diff
@@ -702,7 +744,7 @@ def to_energy_dispersion(self, offset, e_true=None, e_reco=None):
                                 e_reco_lo=ereco_lo, e_reco_hi=ereco_hi,
                                 data=rm)
 
-    def get_response(self, offset, e_true, e_reco=None):
+    def get_response(self, offset, e_true, e_reco=None, mig_step=5e-3):
```

Comment:
I think we call it `migra` pretty consistenly everywhere (also in the spec, and in Gammapy).
So suggest to change `mig_step` to `migra_step` here.